widgets/SpeechToText-transform.json

```diff
@@ -62,6 +69,23 @@
               "en-US"
             ],
             "default" : "en-US"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Transcript Parts Field",
+          "name": "transcriptionPartsField",
+          "widget-attributes" : {
+            "placeholder" : "Field name to use to store all the different chunks of transcription",
+            "default" : "parts"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Transcription Text Field",
+          "name": "transcriptionTextField",
+          "widget-attributes" : {
+            "placeholder" : "Field name to store the transcription with highest confidence score."
```

Comment:
Can't comment below, but this is using the wrong widget. multiple-inputs should be false. Also, don't label it 'Extract Schema', nothing else calls it that, everything else uses 'Get Schema'